projects/opentrain/israel_railways_pipeline/CountLines.java

```diff
@@ -1,26 +1,88 @@
 package projects.opentrain.israel_railways_pipeline;
-
-import com.google.common.collect.ImmutableList;
-
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+import java.io.FileOutputStream;
 import java.io.IOException;
+import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import com.projects.opentrain.israel_railways_pipeline.Proto.Routes;
 
 public class CountLines {
-  public static void main(String[] args) throws IOException {
-    ImmutableList<Path> paths =
-        ImmutableList.of(
-            Paths.get(args[0]), Paths.get(args[1]), Paths.get(args[2]), Paths.get(args[3]));
-    Path outfile = Paths.get(args[4]);
-    StringBuilder result = new StringBuilder();
-    for (Path path : paths) {
-      result.append(path.toString());
-      result.append(":");
-      result.append(Files.readAllLines(path).size());
-      result.append('\n');
+
+  private static final String ISRAEL_RAILWAYS_AGENCY_ID = "2";
+
+  public static List<Routes> getRoutes(Path csvPath) {
+    List<Routes> routes = new ArrayList<>();
+    try {
+      CSVParser parser =
+              CSVParser.parse(
+                      csvPath,
+                      StandardCharsets.UTF_8,
+                      CSVFormat.DEFAULT.withHeader(
+                              "id",
+                              "stop_ids"));
+      boolean isHeader = true;
+      for (CSVRecord record : parser) {
+        if (isHeader) {
+          isHeader = false;
+          continue;
+        }
+        if (!record.get("agency_id").equals(ISRAEL_RAILWAYS_AGENCY_ID)) {
+          continue;
+        }
+        Routes.Builder route = Routes.newBuilder();
+        routes.add(
+                route
+                        .setId(Integer.parseInt(record.get("id")))
+                        .addStopIds(Integer.parseInt(record.get("stop_ids")))
```

Comment:
Did you run the code?
I suppose we will get an exception in this line. Because stop_ids value is an array of integer. I'd advise creating a private method that will get an array of integers and return list of Integers. 

It can look something like:
```
  private static ImmutableList<Integer> parseStopIds(String stopIds) {
    ImmutableList.Builder<Integer> result = ImmutableList.builder();
        // parse `stopIds` and add each value as item to `result`
    return result.build();
  }
```

After that, you can use `parseStopIds()` method to add all `stop_ids`:
```
addAllStopIds(parseStopIds(record.get("stop_ids")))
```
instead of
```
addStopIds(Integer.parseInt(record.get("stop_ids")))
```

Reply:
method replaced (to ImmutableList<Integer> parseStopIds) ,in new PR